src/Components/FeelingsLog/FeelingsLog.css

```diff
@@ -0,0 +1,28 @@
+.feelingsLog {
+    --border-color: rgb(222, 226, 230);
+    --background-color: rgba(0, 0, 0, 0.05);
+    padding: 1.5rem 2rem;
+    border-radius: 6px;
+    border: var(--border-color) 1px solid;
+    width: fit-content;
+    margin: 2rem auto 0;
+}
+
+.feelingsLog table {
+    border-collapse: collapse;
+}
+
+.feelingsLog td,
+.feelingsLog th {
+    padding: .7rem;
+    border-bottom: 1px solid var(--border-color);
+}
+
+th {
+    font-weight: 700;
+    text-align: left;
+}
+
+.feelingsLogs tbody tr:nth-of-type(odd) {
```

Comment:
would love to hear a little about how this line is working: table body, table row, all odd rows have a certain background color?